io.github.archisman_panigrahi.typhoon.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "io.github.archisman_panigrahi.typhoon",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "48",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-version" : "48",
+    "command" : "typhoon",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
+        "--talk-name=org.freedesktop.DBus"
+    ],
+    "build-extensions": [
+        "org.gnome.Platform.Extension.Python3-dbus"
+    ],
+    "modules" : [
+        {
+            "name": "dbus-python",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz",
+                    "sha256": "ad67819308618b5069537be237f8e68ca1c7fcc95ee4a121fe6845b1418248f8"
+                }
+            ]
+        },
+        {
+            "name": "typhoon",
+            "buildsystem": "meson",
+            "config-opts": [
+                "--prefix=/app",
+                "--libdir=lib"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/archisman-panigrahi/typhoon.git",
+                    "tag": "v0.9.86"
```

Comment:
please add a commit here (that match the tag).

Reply:
Fixed